firecrawl-agent/app.py

```
@@ -0,0 +1,254 @@
+            st.stop()
+
+# Main chat interface
+col1, col2 = st.columns([6, 1])
+
+with col1:
+    # Removed the original header
+    st.markdown("<h2 style='color: #0066cc;'>⚙️ Corrective RAG agentic workflow</h2>",
+                unsafe_allow_html=True)
+    # Replace text with image and subtitle styling
+    st.markdown("<div style='display: flex; align-items: center; gap: 10px;'><span style='font-size: 28px; color: #666;'>Powered by LlamaIndex</span><img src='data:image/png;base64,{}' width='50'></div>".format(
+        base64.b64encode(open("./assets/images.jpeg", "rb").read()).decode()
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)